config/default/common/config/metadata/layers/mls/MLS_HNO3_46hPa_Night.md

```diff
@@ -2,4 +2,5 @@ The Microwave Limb Sounder (MLS) Nitric Acid (46hPa, Night) layer shows nitric a
 
 The MLS Nitric Acid (HNO3) Mixing Ratio at 46hPa layer is derived from the MLS Nitric Acid  (ML2HNO3_NRT) available from the Microwave Limb Sounder (MLS) instrument on the Aura satellite. The sensor resolution is 5 km, imagery resolution is 2 km, and the temporal resolution is daily.
 
-References: [ML2HNO3_NRT](https://disc.gsfc.nasa.gov/datasets/ML2HNO3_NRT_004/summary)
\ No newline at end of file
+References: [ML2HNO3_NRT](https://disc.gsfc.nasa.gov/datasets/ML2HNO3_NRT_005
+/summary)
```

Comment:
There is a break in the URL here which breaks the hyperlink. Just remove that and it should be good.

Reply:
ack. i thought i had fixed it, but i guess i forgot to push the change.